packages/docz-tools/src/gatsby-theme-docz/theme/typography.ts

```diff
@@ -1,6 +1,6 @@
 export const fonts = {
-  body: `'Source Sans Pro', sans-serif`,
-  heading: `'Poppins', sans-serif`,
+  body: `'Inter', sans-serif`,
```

Comment:
can we use the `var(--typography--fontFamily-normal)` in here for the body? I know these docz files are finicky so this is an earnest question 😄

Reply:
Let me double check that because docz is super finicky

Reply:
@chris-at-jobber sorry for the delay. I was able to remove it. Just waiting on a build now